src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -51,9 +51,6 @@ public EditCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_JOB).isPresent()) {
             editPersonDescriptor.setJob(ParserUtil.parseJob(argMultimap.getValue(PREFIX_JOB).get()));
         }
-        if (argMultimap.getValue(PREFIX_TAG).isPresent()) {
-            editPersonDescriptor.setTags(ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG)));
```

Comment:
Good, correct implementation - remove parse on Tag since it should be the same anyway